run/django/requirements.txt

```
@@ -1,6 +1,6 @@
-Django==5.1.1; python_version >= "3.10"
-Django==4.2.16; python_version >= "3.8" and python_version < "3.10"
-Django==3.2.25; python_version < "3.8"
+Django==5.1.2; python_version >= "3.10"
+Django==5.1.2; python_version >= "3.8" and python_version < "3.10"
```

Django 5x is not supported on Python 3.8 - 3.9. https://docs.djangoproject.com/en/dev/faq/install/#what-python-version-can-i-use-with-django

```suggestion
Django==4.2.16; python_version >= "3.8" and python_version < "3.10"
```